contracts/core/Registry.sol

```diff
@@ -1,18 +1,33 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
+import { OwnableRoles } from "../../lib/solady/src/auth/OwnableRoles.sol";
+import { Metadata } from "./libraries/Metadata.sol";
+
 // use Solady Roles for ownership of projects: https://github.com/Vectorized/solady/blob/main/src/auth/OwnableRoles.sol
-contract Registry {
+contract Registry is OwnableRoles {
+
+    uint private identityId;
+
     struct IdentityDetails {
         string name;
-        string metadata; // @todo update to MetaPtr
+        Metadata metadata;
         address attestationAddr;
+        // address coreOwner; // ASK: Should this be added?
     }
 
-    // getter for projects mapping
-    function identities(uint _identityId) external view returns (IdentityDetails memory) {
+    mapping(uint => IdentityDetails) public identities;
```

Comment:
Would it make sense to have identities as a public mapping instead and not having the identities getter function ?

Reply:
Yes, agreed.